docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
   portainer:
     image: portainer/portainer
     container_name: "portainer-app"
+#Automatically choose 'Manage the Docker instance where Portainer is running' by adding <--host=unix:///var/run/docker.sock> to the command
```

Comment:
Are you planning to leave that comment?

Reply:
I think it is useful as a 'first start hint', for users who want to run a demo locally, because it avoids a configuration page where they might not be sure about what to choose. After that first run, they can check the docs to better fit their needs.
However, it can be removed if you think it might lead to misunderstanding.